lib/metrics/processMaxFileDescriptors.js

```diff
@@ -25,14 +25,33 @@ module.exports = (registry, config = {}) => {
 			return;
 		}
 
-		fs.readFile('/proc/sys/fs/file-max', 'utf8', (err, maxFds) => {
+		fs.readFile('/proc/self/limits', 'utf8', (err, limits) => {
 			if (err) {
 				return;
 			}
 
+			const lines = limits.split('\n');
+
+			let maxFds;
+			lines.find(line => {
+				const parts = line.split(/  +/);
+				if (parts[0] === 'Max open files') {
```

Comment:
You could save most of the `split` operations if you instead do something like `if (line.startsWith('Max open files')) { line.split(/\s+/)... }`

Reply:
I pushed your suggestion. PTAL. Note that this is called only once during process lifetime.

Reply:
Heh, good point. Thanks for making the change anyway :)